integration/flink/src/test/resources/events/expected_failed.json

```diff
@@ -0,0 +1,3 @@
+{
+  "eventType" : "FAIL"
```

Comment:
Could we include some error message from the caught throwable within an event?

Reply:
Will do in future PR after https://github.com/OpenLineage/OpenLineage/pull/897

I think there's no point in writing code that won't even get released.